src/obsidian-components/search-and-replace-modal.tsx

```diff
@@ -40,15 +40,27 @@ export class SearchAndReplaceModal extends Modal {
 			eventBridge.onArrowDown?.(e, ctx);
 		});
 
-		// Replace note at selectedIndex
+		// Open selectedIndex note
 		this.scope.register([], "Enter", (e, ctx) => {
 			e.preventDefault();
 
 			// Prevent press and hold
 			if (e.repeat) return;
 
+			this.close();
```

Comment:
Please remove this line as we do not want to close the modal if the action is replace